hmy/api_backend.go

```diff
@@ -185,3 +198,9 @@ func (b *APIBackend) GetPoolTransactions() (types.Transactions, error) {
 	}
 	return txs, nil
 }
+
+// GetBalance returns balance of an given address.
+func (b *APIBackend) GetBalance(address common.Address) (*hexutil.Big, error) {
+	balance, err := b.hmy.nodeAPI.GetBalanceOfAddress(address)
+	return (*hexutil.Big)(balance), err
```

Comment:
is there any risk (balance) be nil?

Reply:
Get balance is handled by ethereum GetBalance. I look around ethereum code if nil it returns Big0 etc... For extremely conservative, we can have that check. But if we do that we would have to handle that many other places.
```
// GetBalance retrieves the balance from the given address or 0 if object not found
func (stateDB *DB) GetBalance(addr common.Address) *big.Int {
	stateObject := stateDB.getStateObject(addr)
	if stateObject != nil {
		return stateObject.Balance()
	}
	return common.Big0
}
```

Reply:
fine